articles/dev-lifecycle/setting-up-env.md

```diff
@@ -13,6 +13,22 @@ The example above uses a simple naming convention to distinguish each environmen
 You can request a [child account](/dev-lifecycle/child-tenants) for use in a development/staging/testing environment. Note that free accounts do not include a child account.
 :::
 
+## Environment Tags

```

Comment:
Rephrase to `Set the Environment` (the link will change, so update any links pointing here)